src/Windowing/Silk.NET.Windowing.Desktop/GlfwWindow.cs

```diff
@@ -746,13 +772,28 @@ private unsafe void InitializeCallbacks()
             _glfw.SetDropCallback(_windowPtr, _onFileDrop);
         }
 
+        /// <summary>
+        /// Checks whether or not Vulkan is supported for this window.
+        /// </summary>
+        /// <remarks>
+        /// This doesn't guarantee that context or surface creation will succeed; consult the GLFW documentation for
+        /// more details.
+        /// </remarks>
         public bool IsVulkanSupported => _glfw.VulkanSupported();
 
+        /// <summary>
+        /// Create a Vulkan surface.
+        /// </summary>
+        /// <param name="instance">The Vulkan instance to create a surface for.</param>
+        /// <param name="allocator">A custom Vulkan allocator. Can be omitted by passing null.</param>
+        /// <typeparam name="T">Allocator type</typeparam>
+        /// <returns>A handle to the Vulkan surface created</returns>
+        /// <exception cref="GlfwException">Thrown if GLFW fails to create a Vulkan surface</exception>

```

Comment:
Was there not any documentation to inherit for this method? If not, these doc comments should be moved to the relevant interface in Windowing.Common instead